src/pages/Settings.tsx

```diff
@@ -297,6 +337,51 @@ export const Settings = () => {
     </>
   );
 
+  const exportKeysAsQrCodePage = (
+    <>
+      <BackButton onClick={() => setPage("main")} />
+      <Show
+        when={shouldVisibleExportedKeys}
+        whenFalseContent={<Text theme={theme}>Timed out. Please try again</Text>}
+      >
+        <ExportKeysAsQrCodeContainer>
+            <QrCode address={exportKeysQrData}/>
+        </ExportKeysAsQrCodeContainer>
+
+      </Show>
+    </>
+  );
+
+
+  const exportKeyOptionsPage = (
+    <Show
+      when={!showSpeedBump}
+      whenFalseContent={
+        <SpeedBump
+          theme={theme}
+          message={speedBumpMessage}
+          onCancel={handleCancel}
+          onConfirm={(password?: string) => handleSpeedBumpConfirm(password)}
+          showSpeedBump={showSpeedBump}
+          withPassword={decisionType === "export-keys" || decisionType === "export-keys-qr-code"}
+        />
+      }
+    >
+      <BackButton onClick={() => setPage("main")} />
+      <SettingsRow
+          name="Download Keys"
+          description="Download your seed, private, and public keys"
+          onClick={handleExportKeysIntent}
+        />
+      <SettingsRow
+          name="Export Keys as QR code"
+          description="Show your keys (private,public & seed) as QR code for wallets to scan & import"
```

Comment:
"Display private keys as QR code for mobile import"

or something similar. I'd like to see this on a single line without wrapping